### PR TITLE
Update validator to support new direct reallocations

### PR DESCRIPTION
**Pull request by DanielVF**
With the new style allocations, we also need to update the strategist action validator.

### CODE REVIEW

File: eagleproject/core/static/core/strategist/strategist_validator.js [hunk 1]
```diff
@@ -68,6 +70,7 @@ const strategistValidator = (function () {
         masterEl.appendChild(innerEl);
       }
     } else {
+      iVault.parseTransaction({ data: data });
```
Comment (sparrowDom) [inline eagleproject/core/static/core/strategist/strategist_validator.js:73]: is there a need for this line outside the try-catch?

Comment (DanielVF) [reply eagleproject/core/static/core/strategist/strategist_validator.js:73]: You are right. Leftover debugging. Sharp eyes.

Comment (DanielVF) [reply eagleproject/core/static/core/strategist/strategist_validator.js:73]: Will fix.

Comment (DanielVF) [reply eagleproject/core/static/core/strategist/strategist_validator.js]: Fixed

**Pull request merged**